core/modules/user/user.pages.inc

```diff
@@ -78,7 +78,7 @@ function user_pass_validate($form, &$form_state) {
     form_set_value(array('#parents' => array('account')), $account, $form_state);
   }
   else {
-    form_set_error('name', t('Sorry, %name is not recognized as a user name or an e-mail address.', array('%name' => $name)));
+    form_set_error('name', t('Sorry, %name is not recognized as a user name or an email address.', array('%name' => $name)));
```

Comment:
Suggestion: change "user name" to "username" to match the rest of the instances"

Reply:
Closing in favour of a new issue for these.